src/v1/item/utils/isKey.ts

```diff
@@ -0,0 +1,3 @@
+import { Property } from 'v1/item'
+
+export const isKey = ({ _key }: Property): boolean => _key
```

Comment:
```suggestion
export const isKeyAttribute = ({ _key }: Property): boolean => _key
```

wdyt?

Reply:
All good for me !

Reply:
Just noticed that I use the naming `property` everywhere. But you're right, `Attribute` is more idiomatic. Is it ok for you if I use `isKeyProperty` in this PR and change `Property` to `Attribute` in the next one ?

Reply:
Sure thing!